ina219.py

```diff
@@ -124,8 +124,8 @@ def __init__(self, shunt_ohms, i2c, max_expected_amps=None,
         log_level -- set to logging.DEBUG to see detailed calibration
             calculations (optional).
         """
-        logging.basicConfig(level=log_level)
```

Comment:
I think we should follow the pattern in the RPi version of this library and check if the logger is already setup, see [RPi ina219.py](https://github.com/chrisb2/pi_ina219/blob/master/ina219.py#L109-L114), which was made in this [PR|https://github.com/chrisb2/pi_ina219/pull/20]?